### PR TITLE
[8.0] [MIG] partner_phone_extension

### PR DESCRIPTION
**Pull request by serpentcs-dev1**
@aheficent  Migrated partner_phone_extension to v8

### CODE REVIEW

File: partner_phone_extension/i18n/partner_phone_extension.pot [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Translation of OpenERP Server.
```
Comment (pedrobaeza) [inline partner_phone_extension/i18n/partner_phone_extension.pot]: Remove this file.

**Review by pedrobaeza — DISMISSED** [review]
README.rst file?

File: partner_phone_extension/README.rst [hunk 2]
```diff
@@ -0,0 +1,55 @@
+.. image:: https://img.shields.io/badge/license-LGPLv3-blue.svg
+   :target: https://www.gnu.org/licenses/lgpl.html
+   :alt: License: LGPL-3
```
Comment (serpentcs-dev1) [inline partner_phone_extension/README.rst]: @pedrobaeza @aheficent Should the license be AGPL or LGPL?

**Comment by serpentcs-dev1** [comment]
Added Readme.rst

File: partner_phone_extension/README.rst (hunk 2 above)
Comment (AaronHForgeFlow) [reply partner_phone_extension/README.rst]: I'd respect the existing license, AGPL

File: partner_phone_extension/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+# Copyright 2017 Serpent Consulting Services Pvt. Ltd.
+#   (<http://www.serpentcs.com>)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Partner Phone Number Extension',
+    'version': '8.0.1.0.0',
+    'category': 'Customer Relationship Management',
+    'summary': 'Partner Phone Number Extension',
+    'author': "Savoir-faire Linux,Odoo Community Association (OCA)",
```
Comment (AaronHForgeFlow) [inline partner_phone_extension/__openerp__.py]: I'd also add the added authors here

**Review by AaronHForgeFlow — COMMENTED** [review]
It seems to me that the travis error in partner_changeset test test_changeset_field_type is unrelated. The error is referred to a boolean field.

**Review by AaronHForgeFlow — APPROVED** [review]
Tested on runbot + code review :+1:

**Comment by AaronHForgeFlow** [comment]
Can you review @gmeficent?

**Review by gmeficent — APPROVED** [review]
Code and runbot functional review :+1:

**Review by antoniocanovas — APPROVED** [review]
Functional Ok

**Review by rafaelbn — APPROVED** [review]
tested

**Pull request merged**